File: trunk/quests/fearplane/Fright.pl
#Spawns an iksar broodling on the death of the golems 75% of the time.

sub EVENT_SIGNAL {
 if($signal == 0) {
  quest::say("Such is the will of Cazic-Thule!");
 }
 elsif($signal == 1) {
  quest::say("We obey!");
 }
}

sub EVENT_DEATH {
  my $a = quest::ChooseRandom(72105,72105,72105,0);
  my $x = $npc->GetX();
  my $y = $npc->GetY();
  my $z = $npc->GetZ();
  my $h = $npc->GetHeading();
  quest::spawn2($a,0,0,$x,$y,$z,$h);
}

#Submitted by: Jim Mills (Gilmore Girls`Is`Awesome`XOXO)

File: quests/fearplane/Fright.lua
--Spawns an iksar broodling on the death of the golems 75% of the time.

function event_signal(e)
	entity_list = eq.get_entity_list();

	if(e.signal == 1) then
		e.self:Say("Such is the will of Cazic-Thule!");
	elseif(e.signal == 2) then
		local mobtypeID =  entity_list:GetMobByNpcTypeID(72003);
		local follow_mob = mobtypeID:GetID();
		eq.follow(follow_mob);
	elseif(e.signal == 3) then
		eq.stop_follow();
	end
end

function event_death_complete(e)
	local a = eq.ChooseRandom(72105,72105,72105,0);
	-- local x = e.self:GetX();
	-- local y = e.self:GetY();
	-- local z = e.self:GetZ();
	-- local h = e.self:GetHeading();
	eq.spawn2(a,0,0,e.self:GetX(),e.self:GetY(),e.self:GetZ(),e.self:GetHeading());
end

--Submitted by: Jim Mills (Gilmore Girls`Is`Awesome`XOXO)
-------------------------------------------------------------------------------------------------
-- Converted to .lua using MATLAB converter written by Stryd
-- Find/replace data for .pl --> .lua conversions provided by Speedz, Stryd, Sorvani and Robregen
-------------------------------------------------------------------------------------------------
